### DISCUSSION TITLE
Installation Problem

### DISCUSSION THREAD

**Discussion by vercetti322** [intelowlproject/IntelOwl#1538] [Q&A]
I have installed python, docker, docker - compose and python dependencies.  But when i run below command, i get this error.                                   
![image](https://user-images.githubusercontent.com/116031920/219549365-a838729f-97e3-42bf-bc6a-cf2172fae122.png)
Pls tell how to resolve this.

**Comment by mlodic** [comment]
if you read the error, you would understand that it is a network connection caused by either your machine or the Docker service. It is not an IntelOwl issue

**Reply by vercetti322** [reply]
I will try to rectify it. Thanks!
